egs/jsut/tts1/run.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 
-# Copyright 2018 Nagoya University (Takenori Yoshimura)
+# Copyright 2018 Nagoya University (Takenori Yoshimura), Ryuichi Yamamoto
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
+# NOTE:
+# To run the script, you will need to manually installl pyopenjtalk.
+# Please see https://github.com/r9y9/pyopenjtalk for how to install the package.
+# Also, you need to install other dependences as follows:
+# pip install nnmnkwii jaconv
```

Comment:
Why don't you move this NOTE into path.sh like:
https://github.com/espnet/espnet/blob/master/egs/chime4/asr1_multich/path.sh

Reply:
Added installation check fo pyopenjtalk.

What should I do for other pip-installable external dependencies? I like the following approaches:

1. Add a comment (as is): you need to run `pip install nnmnkwii jaconv`
2. Add `pip install nnmnkwii jaconv` somewhere. 

To be honest I'm not a big fan of writing shell scripts to check pip-installable dependency... If we can take one of the approaches above, we just only needs a single line, which is very simple (and that's why I like it).

Reply:
One of the advantage to add installation check in `path.sh` is that we can detect the installation necessity before starting the stages. (it failed in `path.sh || exit 1;`)
I think this is more user friendly than it fails at the middle of stages.

In the case of pip installable libraries, we can add them in `setup.py`.
Maybe here what we have to check is `openjtalk` installation, which is not easy to install.
(Pyopenjtalk cannot be installed before the installation of openjtalk, right?)
The best solution is adding like `openjtalk.done` which including all of the installations (openjtalk, or other pip installable libraries). But it is optional.

Reply:
I agree that run.sh shouldn't stop in the middle of stages, but I thought we could live with just notifying users to install some additional packages *before* running the script (e.g. adding a notice to README for example). Anyway, I added `pyopenjtalk.done` in Makefile by following the espnet-style.

Reply:
Cool!
Yeah, I agree with you. If we put README in each recipe, that is perfect.